code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -395,6 +395,24 @@
 			SetLuminosity(1)
 			valid = 1
 		*/
+		if(/obj/item/clothing/head/alien_antenna)
+			name = "Al-Ian"
```

Comment:
>AI-Ian
>Not A-Ian

ONE FUCKING JOB

Reply:
I don't get it

Reply:
It's AL-IAN though

Reply:
It's not the same A sound, but you kinda lose the point if you make it a-ian